libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -753,131 +730,6 @@ string ofSystemTextBoxDialog(string question, string text){
 		 DestroyWindow(dialog);
 		 EnableWindow(WindowFromDC(wglGetCurrentDC()), TRUE);
 
-	#else
-
```

Comment:
Oh, and by the way, don't let the indentation fool you: this `#else` and removed code below belongs to `#ifdef _MSC_VER` line, the indentation mixes tabs and spaces.